core/templates/dev/head/pages/exploration_player/CorrectnessFooterDirective.js

```diff
@@ -23,14 +23,6 @@ oppia.directive('correctnessFooter', [
       scope: {},
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/pages/exploration_player/' +
-        'correctness_footer_directive.html'),
-      controller: [
-        '$scope',
-        function($scope) {
-          $scope.correctnessfooterUrl = 
-            UrlInterpolationService.getStaticImageUrl(
-              '/general/footer_tick.png');
```

Comment:
Delete the image if it's not being used anywhere?

Reply:
He did that I think. assets/images/general/footer_tick.png is deleted.

Reply:
Oh sorry! I missed that. Thanks a lot @nithusha21!